src/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/open-lambda/open-lambda/ol
 
-go 1.18
+go 1.21
```

Comment:
If we want to bump the version (good idea, btw!), let's do that in a PR by itself.  That PR should also update the versions used by the tests: https://github.com/open-lambda/open-lambda/tree/main/.github/workflows.

We should check the docs, too (though it looks like that is already on 1.21.5: https://github.com/open-lambda/open-lambda/blob/main/docs/worker/getting-started.md).